__docs__/wonder-blocks-timing/with-action-scheduler.mdx

```diff
@@ -37,7 +37,7 @@ The added `schedule` prop is of type [`IScheduleActions`](#ischeduleactions). Th
 The returned value from `withActionScheduler` is a React component with props of
 type `TProps`.
 
-Access to the [timing API](/docs/timing-types-ischeduleactions--page) is provided via the `withActionScheduler` higher order
+Access to the <a href="./?path=/docs/packages-timing-types-ischeduleactions--docs">timing API</a> is provided via the `withActionScheduler` higher order
```

Comment:
**suggestion:** It looks like this link is broken. It's missing `withactionscheduler`

```suggestion
Access to the <a href="./?path=/docs/packages-timing-withactionscheduler-types-ischeduleactions--docs">timing API</a> is provided via the `withActionScheduler` higher order
```

Reply:
Ah, yeah, that must've broken with a rename at some point.

Reply:
Ah, the issue is that the exported type shouldn't be nested under `withActionScheduler` like that. The `Types` folder should be at the root level of the package docs like with other packages. I'll fix that.